core/Pos/Util/Util.hs

```diff
@@ -31,7 +31,9 @@ module Pos.Util.Util
        , ether
        , Ether.TaggedTrans
        , HasLens(..)
+       , HasLens'
```

Comment:
1. I didn't hate it completely, only in many cases (but not all).
2. It's consistent with the naming used in `ether` which I am already used to.